controllers/useraccount/useraccount_controller.go

```diff
@@ -484,6 +500,93 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	return true, nil
 }
 
+// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console is deleted, `false` otherwise with the underlying error.
```

Comment:
It can happen that there is nothing to delete. And it's not an error. Correct?
Basically it's the following:
- `return true, nil`. Deleted w/o error.
- `return false, nil`. Nothing to delete.
- `return false, err`. Tried to delete existing resources but failed.

Right?
```suggestion
// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console are deleted, `false` otherwise.
```

Reply:
fixed the comment in afd558d